Tests/Docker/Container/ProcessTest.php

```diff
@@ -77,41 +77,14 @@ public function testFilterBrokenUnicode(): void
 fwrite(STDERR, substr('b😀c', 0, 3) . PHP_EOL);
 PHP
         );
+        $outputFilter = new OutputFilter(10000);
         $process = new Process(['php', $this->temp->getTmpFolder() . '/run.php']);
+        $process->setOutputFilter($outputFilter);
         $process->run();
         self::assertSame('a', $process->getOutput());
         self::assertSame('b', $process->getErrorOutput());
     }
 
-    public function testFilterBrokenUnicodeWithCallback(): void
-    {
-        file_put_contents($this->temp->getTmpFolder() . '/run.php', <<<'PHP'
-<?php
-
-fwrite(STDOUT, substr('a😀b', 0, 3) . PHP_EOL);
-fwrite(STDERR, substr('b😀c', 0, 3) . PHP_EOL);
-PHP
-        );
-        $events = [];
-        $process = new Process(['php', $this->temp->getTmpFolder() . '/run.php']);
-        $process->run(function ($type, $e) use (&$events) {
-            return $events[$type][] = $e;
-        });
-        self::assertEquals(
-            [
-                'out' => [
-                    'a',
-                ],
-                'err' => [
-                    'b',
-                ],
-            ],
-            $events
-        );
-        self::assertSame('a', $process->getOutput());
-        self::assertSame('b', $process->getErrorOutput());
-    }
```

Comment:
proc vyhodit tenhle test?

Reply:
Protože je to teď pokrytý přes OutputFilterTest a v ProcessTest se testuje už jen jestli se ten outputFilter zavolá za všech okolností